packages/@aws-cdk/aws-apigatewayv2/lib/http/api-mapping.ts

```diff
@@ -0,0 +1,92 @@
+import { Construct, Resource } from '@aws-cdk/core';
+import { CfnApiMapping, CfnApiMappingProps } from '../apigatewayv2.generated';
+import { IApiMapping, IDomainName } from '../common';
+import { IHttpApi } from '../http/api';
+import { IHttpStage } from './stage';
+
+/**
+ * Properties used to create the HttpApiMapping resource
+ */
+export interface HttpApiMappingProps {
+  /**
+   * Api mapping name
+   * @default -  logical id
+   */
+  readonly apiMappingName?: string;
+
+  /**
+   * Api mapping key
+   * @default -  empty api mapping key
+   */
+  readonly apiMappingKey?: string;
```

Comment:
What does this default mean? Does it mean that this mapping will be on the root path of the domain, or something else? Please clarify in the documentation.

I'm wondering if we make this attribute mandatory instead?

Reply:
OK. Mandatory now :-) Also updated the README for the description.